loky/process_executor.py

```diff
@@ -149,6 +149,67 @@ def clear(self):
                 self._reader.recv_bytes()
 
 
+def _viztracer_init(init_kwargs):
+    """Initialize viztracer's profiler in worker processes"""
+    from viztracer import VizTracer
+    tracer = VizTracer(**init_kwargs)
+    tracer.register_exit()
+    tracer.start()
```

Comment:
The `process_executor.py` file is already very large, I would rather put all this logic in a separate file.